# List exiting control groups handled by the manager

----
ControlGroup[] groups